File: 백준/Bronze/2444. 별 찍기 － 7/별 찍기 － 7.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		
		for(int i=1; i<=2*n-1; i++) {
			for(int j=0; j<Math.abs(i-2*n/2); j++) {
				System.out.print(" ");
			}
			for(int j=0; j<2*(n-Math.abs(i-2*n/2))-1; j++) {
				System.out.print("*");
			}
			System.out.println();
		}

		
	}

}

File: 백준/Bronze/2444. 별 찍기 － 7/별 찍기 － 7.py
n = int(input())


for i in range(1,2*n):
    for j in range(abs(i-2*n//2)):
        print(" ", end="")
    for k in range(2*(n-abs(i-2*n//2))-1):
        print('*', end="")
    print()